src/runtime/module_importer.js

```diff
@@ -12,11 +12,9 @@ exports.initTracer = function () {
     // the version provided by the layer
     const path = require.resolve("dd-trace", { paths: ["/var/task/node_modules", ...module.paths] });
     // tslint:disable-next-line:no-var-requires
-    const tracer = require(path).init({
-        tags: {
-            "_dd.origin": "lambda",
-        },
-    });
+    // add lambda tags to DD_TAGS environment variable
+    const ddtags = updateDDTags({"_dd.origin": "lambda"})
```

Comment:
Name might differ with the current existing function used in metrics, but might be a good think to refactor in the future